commands/commands.go

```diff
@@ -218,6 +222,28 @@ func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclu
 	return includePaths, excludePaths
 }
 
+// help is used for 'git-lfs help <command>'
+func help(cmd *cobra.Command, args []string) {
+	if txt, ok := ManPages[cmd.Use]; ok {
+		fmt.Fprintf(os.Stderr, txt)
+	} else {
+		fmt.Fprintf(os.Stderr, "Sorry, no help text found\n")
+	}
```

Comment:
Weird, it worked fine for me & matched up with the man content; 'Use' is pretty consistently set like this:
```
	pushCmd = &cobra.Command{
		Use:   "push",
```
But I guess arguably someone could make a mistake on setting those up so args would be better.